src/initFolders.ts

```diff
@@ -0,0 +1,35 @@
+import {makeSurePathExists} from './core/utils';
```

Comment:
Please put this file to `src/util`. We'll move all the utility function there soon.